pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -58,19 +59,20 @@ stdenv.mkDerivation (finalAttrs: {
     dconf
     ddcutil
     glib
-    libglvnd
     libpulseaudio
-    libxcb
-    libXrandr
     networkmanager
     ocl-icd
     opencl-headers
     pciutils
     rpm
+    zlib
+  ] ++ lib.optionals x11Support [
+    libXrandr
+    libglvnd
+    libxcb
     vulkan-loader
     wayland
     xfce.xfconf
```

Comment:
this needs to be wrapped in a linux check.. it's broken on darwin.

Reply:
Fixed.